ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/JTreeFixture.kt

```diff
@@ -61,3 +69,17 @@ class JTreeFixture(
         }
     }
 }
+
+fun JTreeFixture.waitUntilLoaded() {
+    step("waiting for loading text to go away...") {
+        try {
+            while (true) {
+                Pause.pause(100)
+                findAllText().any { it.text.equals("loading...", ignoreCase = true) }
```

Comment:
IIRC we had some trees that override the default loading text

Reply:
We do, this should either be overridable probably

Reply:
Which trees?

Reply:
Seems to only be tables for now, so we can think about this later if we need to